File: backend/src/main/java/com/BillingApp/services/CashBankEntryService.java
package com.BillingApp.services;


import java.util.List;

import com.BillingApp.DTO.CashBankEntryDTO;
import com.BillingApp.model.CashBankEntry;

public interface CashBankEntryService {
    CashBankEntry createEntry(CashBankEntry entry);
    List<CashBankEntry> getAllEntries();
    CashBankEntry getEntryById(Long id);
    void deleteEntry(Long id);
    CashBankEntryDTO updateEntry(Long id, CashBankEntryDTO dto);

}

File: frontend/src/services/CashBankEntryService.js
// src/services/CashBankEntryService.js

import axios from "axios";

const API_URL = "http://localhost:8087/api/cashbank";

export const getAllEntries = () => axios.get(API_URL);
export const createEntry = (entry) => axios.post(API_URL, entry);
export const deleteEntry = (id) => axios.delete(`${API_URL}/${id}`);
export const getEntryById = (id) => axios.get(`${API_URL}/${id}`);
